### PR TITLE
Increase unit tests code coverage controller

### PR DESCRIPTION
**Pull request by sudhirverma**
Increase unit tests code coverage controller

### CODE REVIEW

**Comment by jrichter1** [comment]
test this

File: test/unit/pages/confirm/controller-test.js [hunk 1]
```diff
@@ -1,31 +1,93 @@
...
+  describe('exit', function() {
+    it('exit closes active window', function() {
+      sandbox.stub(electron.remote.currentWindow);
+      confirmcontroller.exit();
+      expect(electron.remote.currentWindow.close).calledOnce;
     });
+  });
 
+  describe('download', function() {
+    it('should able to check download', function() {
```
Comment (dgolovin) [inline test/unit/pages/confirm/controller-test.js]: Better name it 'should open url in browser using electron.shell.openExternal' and use test/mock/electron.js in test.

File: test/unit/pages/confirm/controller-test.js [hunk 2]
```diff
@@ -1,31 +1,93 @@
...
+      expect(confirmcontroller.download('url')).to.be.equal(undefined);
```
Comment (dgolovin) [inline test/unit/pages/confirm/controller-test.js]: please do not suggest tests like this of methods that have no return statements, it always passes without errors and adds a little value to tests.

File: test/unit/pages/confirm/controller-test.js [hunk 3]
```diff
@@ -1,31 +1,93 @@
...
+    });
+  });
+
+  describe('isConfigurationValid', function() {
+    it('should able to check Configuration Valid', function() {
```
Comment (dgolovin) [inline test/unit/pages/confirm/controller-test.js]: You mean `should BE able to check configuration IS valid` to be correct, but it still adds nothing to understanding what is checked. It should sound like specification:
* should return true if nothin is detected and all components have been selected for installation;
* should return true if all detected components have supported versions;
* should return false if one of detected components have unsupported version;
* you name it and get coverage as high as you can! :)

**Comment by dgolovin** [comment]
released with https://github.com/redhat-developer-tooling/developer-platform-install/pull/511.

**Pull request closed without merging**